### PR TITLE
Support hostname instead of IP for ltorrent

### PR DESCRIPTION
**Pull request by krumstein**

### CODE REVIEW

**Comment by hicham-a** [comment]
LGTM

**Comment by kostty** [comment]
looks good

**Comment by kostty** [comment]
Jenkins test this please

File: bin/ltorrent [hunk 1]
```diff
@@ -411,7 +412,7 @@ def start_torrent_client():
         settings.ssl_listen = 0
 
         t_session.set_settings(settings)
-        t_session.listen_on(portmin, portmax, ip)
+        t_session.listen_on(portmin, portmax, socket.gethostbyname(ip))
```
Comment (dchirikov) [inline bin/ltorrent]: What will happen if DNS is broken or `ip` points to external interface, not the provisioning one? Worth to add some checks

**Pull request closed without merging**